src/main/resources/db/migration/V1__create_database.sql

```diff
@@ -1 +0,0 @@
-CREATE DATABASE IF NOT EXISTS Job_Role_Manager_Prod_ThomasP;
```

Comment:
Deploy doesn't seem to have run for this branch - this is going to break because the db in prod will have had this sql script run. I think we're going to need to ask Shaun to drop the db. I don't think he's going to like that 😆

Reply:
I think the deploy wasn't running because my branch wasn't up to date with main so I didn't have that GitHub action yet. I am up to date now and it's passing the deploy action